### PR TITLE
113 create script to publish custom metrics to cloudwatch

### PR DESCRIPTION
**Pull request by dbonser**
*Issue #, if available:* 113

*Description of changes:* Added custom-metrics.py and README.md.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by gottumuk** [comment]
I have reviewed and tested the code. @tmcallaghan can your merge with master

File: monitoring/custom-metrics/custom-metrics.py [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+
+    parser.add_argument('--index_count',
+                        action='store_true',
+                        help="log collection index count")
+
+    parser.add_argument('--index_size',
+                        action='store_true',
+                        help="log collection index size")
+
+    args = parser.parse_args()
```
Comment (tmcallaghan) [inline monitoring/custom-metrics/custom-metrics.py:313]: Add Python 3.7 minimum check with optional opt-out as in fl-multiprocess.py and other scripts in this repo.

**Review by tmcallaghan — CHANGES_REQUESTED** [review]
Please add minimum Python version check as per other scripts in the repo. Note this minumum version and the optional opt-out parameter in the readme.

File: monitoring/custom-metrics/custom-metrics.py (hunk 1 above)
Comment (dbonser) [reply monitoring/custom-metrics/custom-metrics.py:313]: Added Python version check

**Pull request merged**